src/Transport/AzureServiceBusTransport.cs

```diff
@@ -110,6 +110,26 @@ public int EntityMaximumSize
         }
         int entityMaximumSize = 5;
 
+        /// <summary>
+        /// The <see cref="TimeSpan"/> idle interval after which topics and queues are automatically deleted.
+        /// </summary>
+        /// <remarks>The minimum duration is 5 minutes. Default value is <see cref="TimeSpan.MaxValue"/>.</remarks>
+        public TimeSpan AutoDeleteOnIdle
+        {
+            get => autoDeleteOnIdle;
+            set
+            {
+                if (value < TimeSpan.FromMinutes(5))
```

Comment:
Worth adding a comment with a link to the docs about the minimal value of `QueueDescription.AutoDeleteOnIdle`.

Reply:
added comment with link to microsoft docs